src/main/java/controllers/MySQLController.java

```diff
@@ -60,6 +60,110 @@ public void editPost(Post newPost) {
             e.printStackTrace();
         }
     }
+    /**
+     * Given an user objects changes all the attributes
+     * such that match the ones given in the parameter
+     * except the id
+     * @param user user object that stores the id to
+     *             update the values
+     */
+    public boolean updateUser(User user){
+        try {
+            String query = "UPDATE FROM `user_info` SET `username`= ?, `password`=?, `bio`=? " +
```

Comment:
Please look at all the attributes of the User object and update the necessary tables in the database. For example, a User has likeHistory. We should be updating the "likes" table in the database in this method, ideally with a helper method. Please start this work ASAP as there is a lot to be done here. I think this should be really similar to addNewUser, if you want to reuse some of the code/helpers from there if it is possible

Reply:
Is it possible that inside users there could be a method that compares two users?

Reply:
What do you mean by "inside users"?